src/main.cc

```diff
@@ -1429,6 +1437,7 @@ ConfigureCurrentKid(const char *processName)
             else
                 TheProcessKind = pkOther; // including coordinator
```

Comment:
BTW, looks like stale/old comment, since 'coordinator' case was processed above.

Reply:
Since the --kid option can be passed via command line, should not we check that its argument is valid? For example, the command `./squid -Cd1 --kid squid` causes an unclear error message:
```
FATAL: Squid is already running: Found fresh instance PID file (/home/ed/work/squid-pool/1/var/run/squid.pid) with PID 6343
    exception location: Instance.cc(121) ThrowIfAlreadyRunningWith
```

Reply:
Fixing that comment is out of scope.

I would not spend _extra_ time on validating values that only Squid should forms. The users have been warned to stay away. Where it is easy to throw on parsing errors, throw, of course, but do not bother spending extra effort to validate every bit.